### PR TITLE
A confirmation dialog

### PR DESCRIPTION
**Pull request by PeterCeresnik**
Hi, I added confirmation dialog when clicking on Remove all (and also on Reset all) as it was one of the listed issues. Translation into different languages than English are needed to be added.

### CODE REVIEW

**Comment by n-apps** [comment]
@PeterCeresnik as for me, it's possible to add this feature in the CountersFragment itself, without calling listener from MainActivity. Second, analytic events should be called after option menu was clicked, not after action was confirmed.

**Comment by PeterCeresnik** [comment]
I moved dialog implementation from MainActivity to CountersFragmet. Still, I kept listener, so the code could be reusable as it is used twice in this fragment. Also analytic event was moved before dialog. I committed changes.

File: app/src/main/java/ua/napps/scorekeeper/counters/CountersFragment.java [hunk 1]
```diff
@@ -107,6 +111,25 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
...
+                    public void onClick(DialogInterface dialog, int id) {
+                        listener.onPositiveButtonClicked(getActivity());
+                    }
+                })
+                .setNegativeButton(R.string.dialog_no, new DialogInterface.OnClickListener() {
+                    public void onClick(DialogInterface dialog, int id) {
+                        dialog.dismiss();
+                    }
+                });
+        alertDialog = builder.create();
```
Comment (n-apps) [inline app/src/main/java/ua/napps/scorekeeper/counters/CountersFragment.java]: you can call `.show()` directly from builder. it's the same code under the hood

File: app/build.gradle [hunk 2]
```diff
@@ -33,19 +34,19 @@ android {
             keyPassword KEY_PASSWORD
         }
     }
-
+  */
```
Comment (n-apps) [inline app/build.gradle]: Pls revert changes for this file

**Comment by PeterCeresnik** [comment]
I made changes you mentioned.

**Pull request merged**